.buildkite/pipeline.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ "${FAKE_RELEASE_TAG}" || "${BUILDKITE_TAG}" ]]; then
+    # Our releases are currently triggered by the existence of a tag
+    echo -e "--- :sparkles: Preparing for a release! :sparkles:"
+
+    if [[ "${FAKE_RELEASE_TAG}" ]]; then
+        echo "Using fake release tag '${FAKE_RELEASE_TAG}'"
+        release="${FAKE_RELEASE_TAG}"
+    elif [[ "${BUILDKITE_TAG}" ]]; then
+        echo "Using release tag '${BUILDKITE_TAG}'"
+        release="${BUILDKITE_TAG}"
+    fi
+    channel="rc-${release}"
+    echo "Release channel is '${channel}'"
+    echo -e "## Habitat Release _${release}_" | buildkite-agent annotate --context "release-manifest"
```

Comment:
Do we expect backslash escapes in `$release`? Just wondering why the `-e` flag for `echo` is being used.

Reply:
@baumanj The `-e` is most likely leftovers from earlier attempts at the pipeline... I can likely remove all of them now.